vehicle/raw_vehicle_cmd_converter/src/csv_loader.cpp

```diff
@@ -66,6 +66,19 @@ bool CSVLoader::validateData(const Table & table, const std::string & csv_path)
   return true;
 }
 
+Map CSVLoader::getMap(const Table & table)
+{
+  Map map = {};
+  for (unsigned int i = 1; i < table.size(); i++) {
```

Comment:
unsigned int is used everywhere.
If there is no reason to use unsigned int, size_t may be better?

Also could you replace `i++` with `++i`. This reduces the number of copy.